rocketmq-connect/rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/service/ClusterManagementServiceImpl.java

```diff
@@ -81,6 +81,14 @@ public void stop() {
         this.defaultMQPullConsumer.shutdown();
     }
 
+    @Override
+    public boolean hasClusterStoreTopic() {
```

Comment:
maybe there's a config  to decide whether to automatically create a topic.

Reply:
Wether automatically create a topic or not depends on the config of broker. I didn't find a mechanism to manually create a topic from the client. Therefore add autoCreateTopicEnable to connector maybe lead to misunderstanding.

Reply:
sorry, I thought you wanted to create a topic in here.